hack/build-operator-podman.sh

```diff
@@ -8,14 +8,13 @@ fi
 IFS=',' read -r -a PLATFORM_LIST <<< "$PLATFORMS"
 
 # Remove any existing manifest and image
-podman manifest rm "${OPERATOR_IMAGE_TAGGED}" || true
-podman rmi "${OPERATOR_IMAGE_TAGGED}" || true
+podman manifest rm "${OPERATOR_IMAGE_TAGGED}" 2>/dev/null || true
+podman rmi "${OPERATOR_IMAGE_TAGGED}" 2>/dev/null || true
```

Comment:
if podman suppress the error in case the images dont exists, please drop the `|| true` 

lets keep the code correct and as minimal as possible please
sorry for the back and forth

Reply:
I think it's better to keep || true, Below is the reason:
Without || true: The command suppresses the error output but will still fail if there's an issue (e.g., manifest not found).
With || true: The command suppresses the error and ensures the script continues even if there's an issue.